src/CMakeLists.txt

```diff
@@ -105,7 +105,7 @@ if (TRUST_STORE_MIR_AGENT_ENABLED)
     core/trust/mir/prompt_main.cpp
   )
 
-  qt5_use_modules(trust-prompt Core Gui Qml Quick)
+  target_link_libraries(trust-prompt Qt5::Core Qt5::Gui Qt5::Qml Qt5::Quick)
```

Comment:
Move these modules inside the same block defined a couple of lines below.

Reply:
No, let those be, it's a minor cosmetic issue (I hope) :-)

Reply:
Well, that is a cosmetic issue in fact, as I tested all of the PRs before sending them :P